<launch>
  <remap from="drone_stateestimation" to="hello"/>
   <include file="$(find tag_position)/launch/add_tags.launch"/>
  <node name="drone_stateestimation" pkg="tum_ardrone" type="drone_stateestimation">
  </node>
  <node name="drone_autopilot" pkg="tum_ardrone" type="drone_autopilot">
  </node>
  <node name="drone_gui" pkg="tum_ardrone" type="drone_gui">
  </node>
  <node name="drone_global_pos" pkg="drone" type="global_position">
  </node>
  <node name="drone_control" pkg="drone" type="control">
  </node>
</launch>
